Dockerfile.riot

```diff
@@ -5,6 +5,6 @@ COPY config.riot.json /app/config.json
 
 # Custom Branding
 RUN rm -rf /app/themes/riot/
-COPY /branding/themes/riot/ /app/themes/
+COPY /branding/themes/riot/ /app/themes/riot/
 RUN rm -f /app/welcome/images/logo.svg
```

Comment:
unrelated to this PR, but isn't this unnecessary given the next line?

Reply:
no, the image needs to be copied to two places